readme.adoc

```diff
@@ -175,15 +170,22 @@ Wait for Istio's components to be ready
 [source,bash]
```

Comment:
So why do we still need to install Jaeger manually? Isn't that the default tracing engine in Istio now?

Reply:
Let me try tomorrow because I am not 100'% sure about that.